api/CSS.json

```
@@ -1317,6 +1317,54 @@
           }
         }
       },
+      "registerProperty": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS/registerProperty",
+          "support": {
+            "chrome": {
+              "version_added": "78"
+            },
+            "chrome_android": {
+              "version_added": "78"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "65"
+            },
+            "opera_android": {
+              "version_added": "65"
```

Opera Android got a little wonky with their version numbers, but since there's no Opera Android version that correlates to Chrome 78 yet, we should mark this as `false`.

```suggestion
              "version_added": false
```